### PR TITLE
CLDR-15385 BRS CLDRModify

### PR DESCRIPTION
**Pull request by btangmu**
-One -fp (DAIP) change for annotations/my.xml

CLDR-15385

- [ ] This PR completes the ticket.

### CODE REVIEW

**Comment by sffc** [comment]
@jslin can you take a look?  Which version of the translation is correct?

**Comment by justinseinlin** [comment]
This is the text label for emoji?

File: common/annotations/my.xml [hunk 1]
```diff
@@ -2007,7 +2007,7 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 		<annotation cp="🚏" type="tts">ဘတ်စ်ကား မှတ်တိုင်</annotation>
 		<annotation cp="🛣">ကားလမ်း | ဟိုင်းဝေး | အမြန်လမ်း</annotation>
 		<annotation cp="🛣" type="tts">ကားလမ်း</annotation>
-		<annotation cp="🛤">ရထားသံလမြး သံလမြး ရထာျး သံလမျး | သံလမ်း</annotation>
+		<annotation cp="🛤">ရထားသံလမွး သံလမွး ရထြား သံလမြး | သံလမျး</annotation>
```
Comment (justinseinlin) [inline common/annotations/my.xml:2010]: I believe the correct version is:

ရထားသံလမ်း | ရထားလမ်း | သံလမ်း

Comment (justinseinlin) [reply common/annotations/my.xml:2010]: I was just guessing `|` is the delimiter for alternate spelling.  Please let me know if that assumption is not correct.

**Comment by macchiati** [comment]
That is correct. The | separates different phrases.

**Comment by justinseinlin** [comment]
> That is correct. The | separates different phrases.
> […](#)

I guess in this case, can we make the change to the commit to have it corrected?  I remember there was another time also trying to correct this emoji but it didn't go anywhere.

**Comment by btangmu** [comment]
It looks like the "correct" version is different from both the "-" and "+" versions for this PR, is that right?

**Comment by btangmu** [comment]
See https://github.com/unicode-org/cldr/pull/1794

**Pull request closed without merging**